src/hooks/core/websocket/types.d.ts

```diff
@@ -6,20 +6,31 @@ export type WebsocketUserDataType = {
   image?: string;
 };
 
+export type BetterSocialMessageType =
+  | 'follow_user'
+  | 'follow_topic'
+  | 'new_topic_post'
+  | 'add_member_to_group'
+  | 'remove_member_from_group'
+  | 'change_channel_detail'
+  | 'leave_group';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The introduction of `BetterSocialMessageType` is a good move towards improving code readability and maintainability. By defining this type, you've made the code more self-explanatory and easier to manage in case of future changes.

```typescript
export type BetterSocialMessageType =
  | 'follow_user'
  | 'follow_topic'
  | 'new_topic_post'
  | 'add_member_to_group'
  | 'remove_member_from_group'
  | 'change_channel_detail'
  | 'leave_group';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->